client/web-sveltekit/src/lib/global-notifications/GlobalNotifications.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+    import {renderMarkdown} from '@sourcegraph/common'
```

Comment:
```suggestion
    import {renderMarkdown} from '$lib/common'
```

`renderMarkdown` is already exported from `$lib/common`.